src/cephadm/cephadm

```diff
@@ -2933,7 +2933,8 @@ def command_bootstrap():
                         ssh_key = '/etc/ceph/ceph.pub'
                         if args.ssh_public_key:
                             ssh_key = args.ssh_public_key.name
-                        out, err, code = call_throws(['ssh-copy-id', '-f', '-i', ssh_key, '%s@%s' % (args.ssh_user, split[1])])
+                        if args.no_ssh_copy_id:
```

Comment:
Probably you want to do: 
`
if not args.no_ssh_copy_id: `
Note: the negation in the name of the option make code more difficult to understand.